packages/astro/src/@types/astro.ts

```diff
@@ -438,6 +438,23 @@ export interface AstroUserConfig {
 	 */
 	site?: string;
 
+
+	/**
+	 * @docs
+	 * @name compressHTML
+	 * @type {boolean}
+	 * @default `true`
+	 * @description
+	 * Set to `true` to enable compressHTML mode. This will remove all whitespace from your final HTMl file, including newlines. This is useful for reducing the size of your final build html bundle
```

Comment:
```suggestion
	 * Astro removes all whitespace from your final HTML file, including line breaks, by default. This is useful for reducing the size of your final build's HTML bundle. To disable this, set the `compressHTML` flag to `false`. 
```
Is "newline" a proper term? I couldn't find a definition, or see this term used anywhere as one word, but more commonly I saw reference to removing "line breaks."  If that is what's meant here, then I think it seems to be a more standard phrase.

Also, it seemed odd to say "set to true" when the default is already true. We should probably standardize this, but right now for other Booleans, we've used "Astro does X by default. Set to false to disable." 


Alternatively, we also have precedence wording like 
> Controls whether or not to remove all whitespace from your final HTML file, including line breaks. This is useful for reducing the size of your final build's HTML bundle.

So, we're not standardized yet (though we should be!) but in either case, if the default is true, we should not be showing/instructing setting to true. If in fact `default: true` is wrong, and it should be false, then the "Controls whether..." wording also works. Or, change the above suggestion to:

> Astro can remove all whitespace from your final HTML file, including line breaks, by default. This is useful for reducing the size of your final build's HTML bundle. To enable this, set the `compressHTML` flag to `true`.